index.js

```diff
@@ -1,10 +1,27 @@
 /* eslint no-process-env:0 */
 const rc = require('rc');
 const merge = require('ut-function.merge');
+const template = require('ut-function.template');
 const path = require('path');
 const fs = require('fs');
 const serverRequire = require;
 
+function interpolate(what = {}, context = {}) {
```

Comment:
We should move this to ut-function.template.
If we pass object, then it will execute this code.

Reply:
done! https://github.com/softwaregroup-bg/ut-function/pull/11